### PR TITLE
news May 2022

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by jeroen** [comment]
I'm rolling out a new feature to show the package readme in the homepage. I think it should be available for all packages by the time this post goes online, so maybe we can include this in the post as well.

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+## rOpenSci HQ
+
+### R-universe prominently displays more information on packages!
+
+Packages on [r-universe.dev](https://r-universe.dev) are now described with even more metadata to help users compare and assess them: count of GitHub stars, (transitive) dependencies, dependent packages and an overall score of the package (similar to pagerank).
+
+{{< figure src="search.jpeg" alt="Package card on R-universe, including a count of github stars, (transitive) dependencies / dependent packages and an overall score of the package (similar to pagerank)" >}}
+
+You will see these numbers both in the search results and in package individual pages.
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:30]: This text seems to imply that I should see those metrics in the screenshot, but I don't... Is the screenshot out of date, or should the text be rephrased a bit? Or, if I've missed the metrics, perhaps a brief description (or edits or arrows or something) to guide the reader.

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+{{< figure src="vroom1.jpeg" alt="Top of a package individual pages featuring the same badges." >}}
+
+Try it out as we'll be finetuning the search ranking in particular!
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:34]:
```suggestion
Try it out as we've been fine-tuning the search ranking as well!
```

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+These metrics are not the only improvement to package individual pages: You might have also noticed (and used!)
+
+-   installation instructions,
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:38]:
```suggestion
-   installation instructions, and,
```

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+{{< figure src="vroom2.jpeg" alt="Installation instructions on a package individual page, ready to be copy-pasted." >}}
+
+-   direct links to reverse dependencies.
+
+{{< figure src="vroom3.jpeg" alt="List of 'users' of a package, that is a list of links to reverse dependencies." >}}
+
+Happy space flying! :rocket: Report any bug or feature request to [our central station](https://github.com/r-universe-org/help).
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:46]: ? (the geek in me must point out that you don't fly in space 😆 )
```suggestion
Happy space travels! :rocket: Report any bug or feature request to [our central station](https://github.com/r-universe-org/help).
```

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+### Next coworking sessions
+
+Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffi LaZerte and various community hosts. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
+
+-   [Tuesday, 07 June 2022 9 AM Australian Western / 1:00 UTC](/events/coworking-2022-06/) "Tackle something you've always wanted to learn" *Hosted by community host Nick Tierney and [Steffi LaZerte](/author/steffi-lazerte/)*
+    -   Cowork on a project of your choice;
+    -   Take this time to tackle something you've always wanted to learn;
+    -   Or talk to [Nick Tierney](https://ropensci.org/author/nicholas-tierney/) and discuss what it is you want to learn, and get some tips and resources to get yourself started!
+
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:56]: After the June coworking, I'm debating putting a hold on them until September, but want to check in with the others first. Ideally if that's settled, we can add a brief note to that effect here, but I'll have to see what's up first.

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
...ng European LTER projects. ReLTER uses the R language to access and interact with the DEIMS-SDR archive of information shared by the Long Term Ecological Research (LTER) network. This package grew within eLTER H2020 as a major project that will help advance the development of European Long-Term Ecosystem Research Infrastructures (eLTER RI - <https://elter-ri.eu>). The ReLTER package functions in particular allow to: - retrieve the information about entities (e.g. sites, datasets, and activities) shared by DEIMS-SDR (see e.g. get_site_info function); - interact with the [ODSEurope](maps.opendatascience.eu) starting with the dataset shared by [DEIMS-SDR](https://deims.org/) (see e.g. [get_site_ODS](https://docs.ropensci.org/ReLTER/reference/get_site_ODS.html) function); - use the eLTER site informations to download and crop geospatial data from other platforms (see e.g. get_site_ODS function); - improve the quality of the dataset (see e.g. get_id_worms). Functions currently implemented are derived from discussions of the needs among the eLTER users community. The ReLTER package will continue to follow the progress of eLTER-RI and evolve, adding new tools and improvements as required.
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:75]: The bullet points in here didn't get formatted correctly. Is that something you want to deal with, or this an automatically created text block?

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### Documentation: HTML \> PDF
+
+This paragraph is not about LaTeX, but about screen-reader users! Please take time to read this [email by Jonathan Godfrey on R-pkg-devel](https://stat.ethz.ch/pipermail/r-package-devel/2022q2/007953.html).
+
+> Please consider the HTML alternative to any pdf, in any context, but especially in anything relating to R.
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:203]: Wow, very powerful and beautifully written email. So much respect for Jonathan Godfrey!

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+Community member Lluís Revilla Sancho also highlighted that [search.r-project.org](https://search.r-project.org/) provides HTML documentation for all CRAN documentation.
+
+### Twelve quick tips for software design
+
+You might enjoy the article ["Twelve quick tips for software design"](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009809) by Greg Wilson. It is not R specific but is relevant to package development. "Design with everyone in mind" can for instance remind of the previous point about HTML documentation.
+
+### Renaming the default branch
+
+If you've been considering renaming the default branch from master (oppressive language) to main, do not miss the Tidyverse blog post Jenny Bryan wrote in October 2021 ["Renaming the default branch"](https://www.tidyverse.org/blog/2021/10/renaming-default-branch/), especially as it features useful usethis functions!
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:213]: Awesome, definitely been on my list of things to do :grin:

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, looks good!

Just a couple of comments regarding the R-universe section and how awesome everything is :grin:

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 1 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:30]: @jeroen could you help with this? (as you meant to replace the screenshots anyway)

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 4 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:46]: Oops!!

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 5 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:56]: Ok, I'll leave as is for now.

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 6 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:75]: It's automatic indeed but I'll leave this comment unresolved so I fix it right before merging, after knitting one last time. Thank you!

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 7 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:203]: :100:

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 8 above)
Comment (maelle) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md:213]: :tada:

**Comment by maelle** [comment]
Thanks so much @steffilazerte!!

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md (hunk 1 above)
Comment (jeroen) [reply content/blog/2022-05-20-ropensci-news-digest-may-2022/index.md]: Thanks, I've updated the screenshots now

File: content/blog/2022-05-20-ropensci-news-digest-may-2022/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,623 @@
...
+
+### Next coworking sessions
+
+Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffi LaZerte  and various community hosts. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.
+
+* [Tuesday, 07 June 2022 9 AM Australian Western / 1:00 UTC ](/events/coworking-2022-06/) "Tackle something you've always wanted to learn" *Hosted by community host Nick Tierney and [Steffi LaZerte](/author/steffi-lazerte/)*
+  * Cowork on a project of your choice; 
+  * Take this time to tackle something you've always wanted to learn;
+  * Or talk to [Nick Tierney](https://ropensci.org/author/nicholas-tierney/) and discuss what it is you want to learn, and get some tips and resources to get yourself started!
+
```
Comment (steffilazerte) [inline content/blog/2022-05-20-ropensci-news-digest-may-2022/index.Rmd:109]:
```suggestion
We're taking a break on Coworking events for July and August, so we'll see you in September!
```

**Pull request merged**